src/controllers/update/update.statement.validation.errors.controller.ts

```diff
@@ -36,22 +38,23 @@ export const post = (req: Request, res: Response, next: NextFunction) => {
     const inNoChangeJourney = !!appData.update?.no_change;
 
     if (!errors.isEmpty()) {
-      return renderPage(res, appData, inNoChangeJourney, formatValidationError(errors.array()));
+      return renderPage(res, appData, inNoChangeJourney, req.body['statementErrorList[]'], formatValidationError(errors.array()));
     }
 
     return req.body[StatementResolutionKey] === StatementResolutionType.CHANGE_INFORMATION
       ? res.redirect(getChangeInformationRedirectUrl(inNoChangeJourney))
-      : res.redirect(getBOStatementsUrl(inNoChangeJourney));
+      : res.redirect(getFirstBOStatementsUrl(inNoChangeJourney));
```

Comment:
how about identifiedBOStatement and ceasedBOStatement?

Reply:
updated - thank you!